packages/daemon/src/app/controller/job.ts

```diff
@@ -121,15 +112,13 @@ export class JobController {
       if (data === null || data === undefined) {
         throw new Error('log not found');
       }
-      successRes(ctx, {
+      this.successRes({
         data: {
```

Comment:
Shall we abstract the data format, too?

Reply:
The data returned should be clear, e.g., if return the job list, it should be an array, else if return one job, it should be a job object.

BTW, no need to review the other controllers, they are not finished.

Reply:
So is this PR WIP?

Reply:
This PR refactored plugin apis, I'm working on the others.
I'm not sure if I should submit them through this PR.